dispatcher/DAQController.py

```diff
@@ -333,7 +259,7 @@ def check_timeouts(self, detector, command = None):
         '''
 
         sendstop = False
-        nowtime = datetime.datetime.utcnow()
+        nowtime = now()
 
         #First check how often we have been timing out, if it happend to often something
         #  bad happend and we start from scratch again
```

Comment:
Line 305 `self.error_stop_count[detector] = 0` works per dectector, however, the hypervisor does not allow the MV nor NV to be controlled, if we hit this, should we break linked mode? Otherwise I foresee an infinite loop since the error count is set to 0

Reply:
`self.error_stop_count` refers to logical detectors, not physical detectors. If the MV or NV are not responding during linked mode, then nothing happens until that situation is rectified.